pkg/compare/compare_test.go

```diff
@@ -508,16 +507,11 @@ func getCommand(t *testing.T, test *Test, modeIndex int, tf *cmdtesting.TestFact
 }
 
 func setClient(t *testing.T, resources []*unstructured.Unstructured, tf *cmdtesting.TestFactory) {
-	resourcesByType, _ := groups.Divide(
-		resources,
-		func(element *unstructured.Unstructured) ([]int, error) { return []int{0}, nil },
-		func(e *unstructured.Unstructured) (*unstructured.Unstructured, error) { return e, nil },
-		createGroupHashFunc([][]string{{"kind"}}),
-	)
-	resourcesByKind := lo.MapKeys(resourcesByType[0], func(value []*unstructured.Unstructured, key string) string {
-		// Converted to URL Path Format:
-		return fmt.Sprintf("/%ss", strings.ToLower(key))
-	})
+	resourcesByKind := make(map[string][]*unstructured.Unstructured)
+	for _, t := range resources {
+		key := fmt.Sprintf("/%ss", strings.ToLower(t.GetKind()))
+		resourcesByKind[key] = append(resourcesByKind[key], t)
```

Comment:
As I understand `MapKeys` also transforms to another type, does it needed here?

Reply:
I think because key and the return value are both strings it doesn't do any type change.

I assume go's type system would have complained if there was an issue.